src/components/Table.module.scss

```diff
@@ -10,6 +10,16 @@ $border-style: 1px solid $border-color;
     text-align: center,
   }
 
+  .skin {
```

Comment:
Matter of preference, I suppose...Since `Table` is a generic component that is used to display different types of info, I’d expect that class names here to be generic as well. So instead of `.skin`, perhaps `.extraInfo` or `.annotation`, which would suggest that it can be used in any context (e.g. A not on the rounds attended column to signify how many rounds are incomplete or something). Same with `.bestScore`, perhaps `.highlight`

Reply:
Good point, this was a bit quick and dirty. I think it'll makes sense to do a broader rethinking/reorganization of how some of some of these styles.

Reply:
Curious to know what you see as the broad issues/limitations of the current setup?

Reply:
Just that I never put a ton of thought into how the styles were organized and applied. Importing styles from a module works, but I don't think it's a great pattern in React, where ideally we'd have some kind of reusable component rather than a CSS class. Tables are a bit trickier though. In any case, none of this is all that high-pri.

Reply:
Ah, I see. Yeah, React Components make more sense. After understanding what the classes were doing, I just created subcomponents of `Table` that encapsulate the different types of cells we want. Check it out here: 89cfff7